test/easyconfigs/easyconfigs.py

```diff
@@ -1107,16 +1106,7 @@ def test_pr_python_packages(self):
                 if use_pip is None and not any(re.match(regex, ec_fn) for regex in whitelist_pip):
                     failing_checks.append("'use_pip' should be set in %s" % ec_fn)
 
-            # download_dep_fail is enabled automatically in PythonBundle easyblock, so shouldn't be set
-            if easyblock in ['CargoPythonBundle', 'PythonBundle']:
-                if download_dep_fail or exts_download_dep_fail:
-                    fail = "'*download_dep_fail' should not be set in %s since PythonBundle easyblock is used" % ec_fn
-                    failing_checks.append(fail)
-                if pure_ec.get('exts_default_options', {}).get('source_urls') == python_default_urls:
-                    failing_checks.append("'source_urls' should not be defined in exts_default_options when using "
-                                          "the default value in %s" % ec_fn)
-
-            elif exts_defaultclass == 'PythonPackage':
+            if exts_defaultclass == 'PythonPackage':
                 # bundle of Python packages should use PythonBundle
                 if easyblock == 'Bundle':
```

Comment:
The failures in https://github.com/easybuilders/easybuild-easyconfigs/actions/runs/7818343578/job/21368323894?pr=19265 are all triggered in the `else` side of this `if` (I cannot comment there directly as it is too far from code being changed).

Do we also need to remove that?

Reply:
fixed in 0fd98988ae